tensorflow/core/util/gpu_device_functions.h

```diff
@@ -141,9 +166,9 @@ namespace detail {
 // On Volta, for some invalid masks, this function hangs or returns false
 // positives, because the implementation shuffles with the same mask that
 // we are validating. Run on Pascal if you suspect that the mask is incorrect.
-__device__ inline bool CudaValidateShuffleSyncMask(unsigned mask,
+__device__ inline bool GpuValidateShuffleSyncMask(unsigned mask,
                                                    unsigned src_lane) {
-  unsigned src_dst_mask = 1u << CudaLaneId() | 1u << src_lane;
+  unsigned src_dst_mask = 1u << GpuLaneId() | 1u << src_lane;
```

Comment:
You do start calling the Gpu variant here and in a number of places below. The PR is still manageable, so let's take this shortcut.

Reply:
Sounds good, thanks